glade_forms/lib/src/validator/validator_result.dart

```diff
@@ -1,22 +1,36 @@
 import 'package:equatable/equatable.dart';
 import 'package:glade_forms/src/core/core.dart';
-import 'package:glade_forms/src/validator/validator_error/glade_validator_error.dart';
+import 'package:glade_forms/src/validator/validator_result/glade_validator_result.dart';
 
 class ValidatorResult<T> extends Equatable {
   final GladeInput<T>? associatedInput;
-  final List<GladeValidatorError<T>> errors;
+  final List<GladeValidatorResult<T>> all;
```

Comment:
Do we need to have `all` while we have `warnings` and `errors`? I think we can do all operations on those?

Reply:
Might also be just a getter if needed?